src/modules/sub-items/components/pagination/pagination.component.js

```diff
@@ -0,0 +1,126 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+
+import PaginationBtnComponent from './pagination.btn.component';
+
+export default class PaginationComponent extends PureComponent {
```

Comment:
It should be a functional component instead.

Reply:
I decided to use class component because with that many functions I think it is more readable and clear. Do you want me to change it to functional component and change each method to arrow function within it?

Reply:
The functions inside the class can be easily converted to a _standard_ arrow functions outside of functional component. They will take props as arguments and return a value.
I believe it's better not to define such functions inside the functional component as they're not very specific.